Football team cards/index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
```

Comment:
This project is having only 1 football team that is argentina in future we should add more fav teams in this project.